tests/Core/Content/Product/SalesChannel/Listing/ProductListingFeaturesSubscriberTest.php

```diff
@@ -1193,11 +1195,17 @@ public function testMultipleListingEventsWillOnlyHandleTheRequestOnce(): void
         $decoratedSubscriberMock = $this->getMockBuilder(ShopwareProductListingFeaturesSubscriber::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $decoratedSubscriberMock->expects($this->once())->method('handleListingRequest');
+        $decoratedSubscriberMock->expects($this->exactly(4))->method('handleListingRequest');
```

Comment:
Add a comment why this should be called 4 times, so it is clear in the future when reading this again.